src/components/Header/Header.tsx

```diff
@@ -204,21 +204,65 @@ export const Header: React.FC = () => {
       </div>
       <div className={style.headerBottom}>
         <div className={style.time}>{localTime}</div>
-        <div className={style.news}>
-          <div className={style.announcements}>
-            <a href="#" className={style.announcementsLink}>
-              <img src="/tv.png" alt="tv" className={style.announcementsIcon} />
-              Announcements
-            </a>
-            <div className={style.announcementsCount}>270</div>
+        <div className={style.headerBottomContainer}>
+          <div className={style.news}>
+            <div className={style.announcements}>
+              <a href="#" className={style.announcementsLink}>
+                <img
+                  src="/tv.png"
+                  alt="tv"
+                  className={style.announcementsIcon}
+                />
+                Announcements
+              </a>
+              <div className={style.announcementsCount}>270</div>
+            </div>
+            <div className={style.msg}>
+              |
+              <a href="#" className={style.msgLink}>
+                Personal Msg
+              </a>
+            </div>
           </div>
-          <div className={style.msg}>
-            |
-            <a href="#" className={style.msgLink}>
-              Personal Msg
+          <div className={style.search}>
+            <div className={style.searchTeam}>
+              <input
+                type="text"
+                className={style.searchInput}
+                placeholder="Search Team"
+              />
+              <div className={style.searchButton}>
+                <img
+                  src="/search.png"
+                  alt="search"
+                  className={style.searchIcon}
+                />
+              </div>
+            </div>
+            <a className={style.headerButton} href="#">
```

Comment:
1) This link should be if it redirects to another website.
2) If it's link on our website then it should be a Link from react router if I am not mistaken
3) But if this element does not do anything and there is some modal or js logic, then it must be a button.